app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -35,6 +35,31 @@ def show
 
     def notes
       redirect_to(action: :show) unless FeatureFlag.active?('notes')
+
+      @new_note_form = ProviderInterface::NewNoteForm.new
+      @notes = @application_choice.notes.order('created_at DESC')
+    end
+
+    def new_note
```

Comment:
`application_choices_controller` is losing focus a bit — what do you think about pulling this out into a separate controller?

Reply:
The main reason not to is the routes are scoped to the application choice and the index view for notes requires the subnavigation components for the application's tabs. It feels a little weird moving e.g. `new_note` and `create_note` to a separate controller while the index view remains within `application_choices_controller`. Ideas?